README.md

```diff
@@ -29,16 +26,17 @@ Make sure to use these settings:
 * **Logzio Host**: Use your Logz.io region's listener URL.
   If your login URL is app.logz.io, use `listener.logz.io` (this is the default setting).
   If your login URL is app-eu.logz.io, use `listener-eu.logz.io`.
-* **Logzio Token**: Use the [token](https://app.logz.io/#/dashboard/settings/general) of the account you want to ship to.
+* **Logs Account Token**: Use the [token](https://app.logz.io/#/dashboard/settings/general) of the logs account you want to ship to.
+* **Metrics Account Token**: Use the [token](https://app.logz.io/#/dashboard/settings/general) of the metrics account you want to ship to.
```

Comment:
@imnotashrimp 
The correct names are: Logzio Logs Host, Logzio Metrics Host, Logzio Logs Token, Logzio Metrics Token

Reply:
What would metrics listener url be?

Reply:
@imnotashrimp 
Logzio Metrics Host - the options are the same as the logs host